rs/nervous_system/agent/src/helpers/nns.rs

```diff
@@ -0,0 +1,158 @@
+use std::time::Duration;
+
+use cycles_minting_canister::{NotifyMintCyclesSuccess, MEMO_MINT_CYCLES};
+use ic_base_types::PrincipalId;
+use ic_nns_common::pb::v1::{NeuronId, ProposalId};
+use ic_nns_constants::CYCLES_MINTING_CANISTER_ID;
+use ic_nns_governance_api::pb::v1::{
+    manage_neuron_response::Command, CreateServiceNervousSystem, MakeProposalRequest,
+    ManageNeuronCommandRequest, ProposalActionRequest, ProposalInfo, Topic,
+};
+use ic_sns_wasm::pb::v1::get_deployed_sns_by_proposal_id_response::GetDeployedSnsByProposalIdResult;
+use icp_ledger::{AccountIdentifier, Subaccount, Tokens, TransferArgs};
+
+use crate::nns::governance::{get_proposal_info, manage_neuron};
+use crate::nns::sns_wasm::get_deployed_sns_by_proposal_id;
+use crate::sns::Sns;
+use crate::{CallCanisters, ProgressNetwork};
+
+// TODO @rvem: we probably need more meaningful error type rather than just 'String'
```

Comment:
The response contains an error type I believe.

Reply:
Could you elaborate a bit?

Reply:
I think @rvem meant by this comment that we serialize the errors into strings, which is good enough for now, but might need to be generalized in the future (since these helpers might be used not just for sns-testing).

@max-dfinity, would it be okay to close this thread to unblock the PR? I'm happy to remove the comments if it's confusing.

Reply:
I don't see any need to update this, other than to try to increase consistency and quality.  I'll leave it to you @aterga

Reply:
I think for this PR it's fine to keep this as-is; a future PR could refine the helper errors.

(I suppose that one day we might decide to publish the helpers for clients to use also outside of this repo, in which case we'd lose the ability to refine the helper APIs so easily. Until then, I believe it's safe so iterate)